website/docs/r/monitor_metric_alert.html.markdown

```diff
@@ -100,6 +107,28 @@ A `criteria` block supports the following:
 
 ---
 
+A `dynamic_criteria` block supports the following:
+
+* `metric_namespace` - (Required) One of the metric namespaces to be monitored.
+* `metric_name` - (Required) One of the metric names to be monitored.
+* `aggregation` - (Required) The statistic that runs over the metric values. Possible values are `Average`, `Count`, `Minimum`, `Maximum` and `Total`.
+* `operator` - (Required) The criteria operator. Possible values are `LessThan`, `GreaterThan` and `GreaterOrLessThan`.
+* `alert_sensitivity` - (Required) The extent of deviation required to trigger an alert. Possible values are `Low`, `Medium` and `High`.
+* `dimension` - (Optional) One or more `dimension` blocks as defined below.
+* `evaluation_total_account` - (Optional) The number of aggregated lookback points. The lookback time window is calculated based on the aggregation granularity (`window_size`) and the selected number of aggregated points.
```

Comment:
typo here, should be `evaluation_total_count`

Reply:
Oh, Thank you!